### ISSUE TITLE
Loss functions

### ISSUE THREAD

**Issue by PeterMcGor** [JoHof/lungmask#42]
Hi all; 

I have been reading the paper and I realised that the loss functions are not explicitly shown. Are you using the original loss functions described as in it architecture references? 

Best!

**Comment by JoHof** [comment]
Hi Peter,

thanks for the comment. You are right we should have stated the loss used. Sorry about that. We used cross entropy

**Comment by PeterMcGor** [comment]
Thanks for the quick answer!

I just load your unet trained model to try TL and see that the last activation layer is LogSoftMax. Could be NLLLoss more appropriate? 

Cheers!

**Comment by JoHof** [comment]
You can either remove the logsoftmax and use cross entropy or nllloss. I guess it won't make much difference. I think I added the logsoftmax for this inference code only and used cross entropy for training

**Comment by PeterMcGor** [comment]
Sure! Just curious about stability and repeatability
Cheers!
